zebrad/tests/common/sync.rs

```diff
@@ -314,6 +314,10 @@ pub fn check_sync_logs_until(
     if check_legacy_chain {
         zebrad.expect_stdout_line_matches("starting legacy chain check")?;
         zebrad.expect_stdout_line_matches("no legacy chain found")?;
+
+        zebrad.expect_stdout_line_matches("starting state checkpoint validation")?;
+        // TODO: what if the mempool is enabled for debugging before this finishes?
```

Comment:
What would be the issue if the mempool is enabled for debugging?

Reply:
Those logs would be discarded while we're still waiting for this line, then the mempool log checks below would fail.

Reply:
Ah, okay. This fn isn't used with `debug_enable_at_height` anywhere currently so a TODO should be sufficient.